apps/web/ui/partners/fraud-risks/commissions-on-hold-table.tsx

```diff
@@ -0,0 +1,164 @@
+import useWorkspace from "@/lib/swr/use-workspace";
+import { CommissionResponse, FraudEventProps } from "@/lib/types";
+import { COMMISSIONS_MAX_PAGE_SIZE } from "@/lib/zod/schemas/commissions";
+import { CustomerRowItem } from "@/ui/customers/customer-row-item";
+import {
+  LoadingSpinner,
+  Table,
+  TimestampTooltip,
+  useTable,
+  useTablePagination,
+} from "@dub/ui";
+import {
+  cn,
+  currencyFormatter,
+  fetcher,
+  formatDateTimeSmart,
+  nFormatter,
+} from "@dub/utils";
+import { useState } from "react";
+import useSWR from "swr";
+import { CommissionTypeBadge } from "../commission-type-badge";
+
+export function CommissionsOnHoldTable({
+  fraudEvent,
+}: {
+  fraudEvent: FraudEventProps;
+}) {
+  const workspace = useWorkspace();
+  const { id: workspaceId, slug } = workspace;
+
+  const [page, setPage] = useState(1);
+  const { pagination, setPagination } = useTablePagination({
+    pageSize: COMMISSIONS_MAX_PAGE_SIZE,
+    page,
+    onPageChange: setPage,
+  });
+
+  const query = {
+    workspaceId: workspaceId!,
+    status: "pending",
```

Comment:
@devkiran for the "Commissions on hold" table I'm currently just fetching pending commissions - if we to include other statuses we may need to update the endpoint to either accept an array or a special flag for on-hold commissions.